CMSgov/design-system
# [WNMGDS-698] Add Spanish translation prop to US banner component

### Added
- Added `locale` prop to display USA Banner in Spanish

### How To test
- Run the doc site locally with `yarn start`
- Then visit the USA Banner component page and ensure the correct text is printed when the Spanish translation prop (`locale = "es"`) is set
